### PR TITLE
feat: change user-provided label format in comments

### PR DESCRIPTION
**Pull request by lachrist**
Before, the agent expected labels as `/* @security @login */`. Now it expects labels as `/* @label security login */` which is more consistent with the other agents.

### CODE REVIEW

File: components/trace/appmap/classmap/estree/visit.test.mjs [hunk 1]
```diff
@@ -70,13 +75,13 @@ assertDeepEqual(
...
-          labels: ["@label-1", "@label-2", "@label-3"],
+          labels: ["label-1", "label-2", "label-3"],
           children: [],
         },
       ],
     },
-  ],
-);
+  ]);
+}
```
Comment (apotterri) [inline components/trace/appmap/classmap/estree/visit.test.mjs:87]: Will you add one more test, please: a function that has a non-empty comment, but that has no labels in it? Thanks! (If that test already exists, and I just missed, let me know.)

**Review by apotterri — CHANGES_REQUESTED** [review]
I think this would benefit from one more simple test, then should be good to go.

File: components/trace/appmap/classmap/estree/visit.test.mjs (hunk 1 above)
Comment (lachrist) [reply components/trace/appmap/classmap/estree/visit.test.mjs:87]: The agent works by collecting an array of leading comments. So the comment at line 63 is per requested. If you feel that is not enough I can add a test case of course. However I tend to rely a lot on coverage and not add test cases which do not further improve coverage. This is to reduce change impact.

```js
// leading comment 1
/* leading comment2 */
function f () {}
```

Comment (apotterri) [reply components/trace/appmap/classmap/estree/visit.test.mjs:87]: So, you're saying that each element in the comments array gets tested as a separate comment, right? If that's the case, I'm satisfied with the tests as-is.

**Review by apotterri — APPROVED** [review]

File: components/trace/appmap/classmap/estree/visit.test.mjs (hunk 1 above)
Comment (lachrist) [reply components/trace/appmap/classmap/estree/visit.test.mjs:87]: Indeed
